pmm-app/src/pmm-qan/panel/components/Details/Explain/Explain.tools.ts

```diff
@@ -80,8 +80,8 @@ export const fetchExplains = async (
       };
 
       const [classicResult, jsonResult] = await Promise.all([
-        mysqlMethods.getExplainTraditional(payload).then(getActionResult),
-        mysqlMethods.getExplainJSON(payload).then(getActionResult),
+        mysqlMethods.getExplainMysql(payload).then(getActionResult),
```

Comment:
The `MySQL` in the method name is probably redundant as the `mysqlMethods` already provides information that the explain methods are for mysql.